augly/image/functional.py

```diff
@@ -2004,6 +2005,13 @@ def resize(
     @param height: the desired height the image should be resized to have. If
         None, the original image height will be used
 
+    @param resample: An optional resampling filter. This can be one of
+        PIL.Image.NEAREST, PIL.Image.BOX, PIL.Image.BILINEAR, PIL.Image.HAMMING,
+        PIL.Image.BICUBIC or PIL.Image.LANCZOS. If the image has mode “1” or “P”,
+        it is always set to PIL.Image.NEAREST. If the image mode specifies a number
+        of bits, such as “I;16”, then the default filter is PIL.Image.NEAREST.
+        Otherwise, the default filter is PIL.Image.BICUBIC.
```

Comment:
```suggestion
        PIL.Image.BICUBIC or PIL.Image.LANCZOS.
```

Looks like you copied the whole description from PIL, but that's not accurate to this function.